.storybook/custom-decorators/tabs.decorator.js

```diff
@@ -0,0 +1,57 @@
+import { useEffect } from "@storybook/client-api";
+
+const lastViewedLangKey = "omrs-storybook:preferred-lang";
+
+function getLastViewedLang() {
+  return localStorage.getItem(lastViewedLangKey);
+}
+
+function getTabs() {
+  return document.querySelectorAll(".tab");
```

Comment:
So we switch all the tabs at once, across all examples? Not just the tab you clicked on.

I guess that's probably a pretty good ux.

Reply:
that's right, switch all tabs at once to the syntax of the tab clicked.